SConstruct

```diff
@@ -343,6 +365,16 @@ for path in [
 		CXXFLAGS = [ "-isystem", path ]
 	)
 
+if env['CXX'] == 'clang++':
```

Comment:
This seems prone to breaking if someone needs to pass a full path to the compiler, or if the compiler has a version number in it. Elsewhere in this SConstruct we've used `if "g++" in os.path.basename( env["CXX"] )`, so perhaps we should do similar here?

Reply:
updated in 3d5465b and updated the Cortex PR: https://github.com/ImageEngine/cortex/pull/700